dashboard/common/utils.py

```diff
@@ -35,7 +35,7 @@ def get_git_version_info():
 
 def look_up_key_for_value(myDict, searchFor):
```

Comment:
Since this is only used by the one view and you've added more code specific to the one use, maybe we can rename the function to be something like `look_up_key_for_resource_value`?